src/routes/__tests__/index.test.js

```diff
@@ -0,0 +1,10 @@
+import { render } from "@testing-library/svelte";
+import Index from "../index.svelte";
+
+describe("index component", () => {
+  test("should render component correctly", () => {
+    const { container } = render(Index);
```

Comment:
This test is not helpful right now. But you already mentioned that if I remember correctly. :smile:

Reply:
whoooat? :D

Reply:
okay, dear Markus: what would be a helpful test? :)

Reply:
From my point of view snapshot tests are helpful if you want to make sure something does not change.
But this isn't the case in this test, because we want to make sure the index component should render correctly. What does correctly mean here isn't obvious.
I would guess checking for at least the key features of the index component would be better instead of a snapshot.

Reply:
But we could do that later on, feel free to just resolve my comment here :wink:

Reply:
I would think so, too :) 

I needed one Jest test to integrate Jest in general. The tests fail if there is not a single test. I merge that PR now and we can discuss the testing of the components afterwards and involve the others